documentation/release_6.3.htm

```diff
@@ -132,6 +132,13 @@ <h3>Changed functionality</h3>
     a <code>shared_ptr&lt;const ProjDataInfo&gt;</code>.<br>
     <a href=https://github.com/UCL/STIR/pull/1315>PR #1315</a>
   </li>
+  <li>
+    As <code>stir::Array</code> template arguments might change in the future,
+    it is now recommended to include <tt>stir/ArrayFwd.h</tt> when using forward
+    declaration and use the <code>ArrayType</code> template-alias in places where
+    a rectangular array that might live on a GPU is intended.<br>
+    <a href=https://github.com/UCL/STIR/pull/1589>PR #1589</a>
+  </li>
```

Comment:
why does this appear as a change in this PR? something wrong with the rebasing?

Reply:
hmm... weird. Let me look into it.